apps/champions/lib/champions/units.ex

```diff
@@ -118,7 +119,7 @@ defmodule Champions.Units do
   def calculate_level_up_cost(unit),
     do: [
       %CurrencyCost{
-        currency_id: Currencies.get_currency_by_name!("Gold").id,
+        currency_id: Currencies.get_currency_by_name_and_game!("Gold", Utils.get_game_id(:champions_of_mirra)).id,
```

Comment:
It is not necessary to address this comment here but I think we should put these strings into a module attribute

Reply:
I think this is a placeholder and i would not change this